bitcoin/src/psbt/mod.rs

```diff
@@ -712,15 +712,17 @@ impl Psbt {
     /// - [`Error::NegativeFee`] if calculated value is negative.
     /// - [`Error::FeeOverflow`] if an integer overflow occurs.
     pub fn fee(&self) -> Result<Amount, Error> {
-        let mut inputs: u64 = 0;
```

Comment:
Something I think core does does well is asks contributors to make atomic commits in CONTRIBUTING.md.  This section does both a refactor and adds changes to `Amount` which makes it more difficult to review.

Reply:
here's an excerpt:

`
Patchsets should always be focused. For example, a pull request could add a feature, fix a bug, or refactor code; but not a mixture. Please also avoid super pull requests which attempt to do too much, are overly large, or overly complex as this makes review difficult.
`

While this project is not core, there's lots of great advice in the Core CONTRIBUTING.md

Reply:
This PR is WIP _and_ in draft state.